packages/frontend/amp/components/Ad.tsx

```diff
@@ -99,15 +121,17 @@ interface CommercialConfig {
     usePrebid: boolean;
 }
 
-export const Ad: React.SFC<{
-    edition: Edition;
-    section: string;
-    contentType: string;
-    config: CommercialConfig;
-    commercialProperties: CommercialProperties;
-}> = ({ edition, section, contentType, config, commercialProperties }) => (
-    <div className={adStyle}>
+const ampAdElem = (
+    adRegionClass: string,
+    edition: Edition,
```

Comment:
`edition`should be redundant now

Reply:
Spoke to Alex and the rest of the team. Edition is still needed for the targetting json. Everywhere else should be replaced with geo location region.